tests/test_select.py

```diff
@@ -0,0 +1,59 @@
+"""Collections of test for perun.select package"""
+
+# Standard Imports
+import git
+import pytest
+
+# Third-Party Imports
+
+# Perun Imports
+from perun import vcs
+from perun.profile import helpers as profile_helpers
+from perun.select.abstract_base_selection import AbstractBaseSelection
+from perun.select.whole_repository_selection import WholeRepositorySelection
```

Comment:
Could we perhaps import both in `select.__init__.py` so that we can import directly from `perun.select`?

Reply:
Will be resolved in next PR.